.pre-commit-config.yaml

```diff
@@ -1,25 +1,15 @@
 repos:
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.2.1
```

Comment:
Could you please move the `ruff` hook to be after the `prettier` hook? This will allow us to have a clearer diff.

Reply:
Done